chain-vote/src/tally.rs

```diff
@@ -156,6 +162,37 @@ impl EncryptedTally {
     }
 }
 
+impl ValidatedTally {
+    /// Given the shares of the committee members, returns the decryption of all the
+    /// election options in the form of `GroupElements`. To get the final results, one
+    /// needs to compute the discrete logarithm of these values, which is performed in
+    /// `decrypt_tally`.
```

Comment:
Good doc; for a moment, I was confused by this being a doc comment, thinking that you are adding a new public method. For private methods, plain comments should be made as a rule.